components/form/ChangePassword.vue

```diff
@@ -39,11 +39,11 @@ export default {
   },
   data(ctx) {
     return {
-      username:                    '',
-      errorMessages:               [],
-      pCanShowMissmatchedPassword: false,
-      pIsRandomGenerated:            false,
-      form:                        {
+      username:                   '',
+      errorMessages:              [],
+      pCanShowMismatchedPassword: false,
+      pIsRandomGenerated:         false,
+      form:                       {
```

Comment:
The whitespace was already broken. Otherwise, this change is local to the template and uninteresting.